$NetBSD: patch-af,v 1.1 2001/08/13 22:27:50 wennmach Exp $

XDestroyImage() frees both the image structure and the data pointed to
by the image structure.

--- xworld.c.old	Tue Aug 14 00:16:41 2001
+++ xworld.c		Tue Aug 14 00:17:03 2001
@@ -466,7 +466,6 @@
                         r1 = n/2;
                         r2 = size/2;
                         XDestroyImage(image);
-                        free(Map);
                         image = XCreateImage(display, visual, depth, format,
                           0, 0, size, size, bitmap_pad, 0);
                         if (image == NULL) {
